pyroengine/platforms/pc/pc_engine.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+import cv2
+import sys
+from PIL import Image
+from pyroengine.engine import PyronearEngine
+
+
+class PcEngine(PyronearEngine):
+    """
+    This class is the Pyronear Engine for pc. This engine manage the whole Fire Detection
+    process by capturing and saving the image and by predicting if there is a fire or
+    not based on this image.
+    Example
+    -------
+    # For a prediction every 5 frame
+    python pi_engine.py api_login, api_password 5
+    """
+    def run(self, every_n_frame=10):
+
+        cap = cv2.VideoCapture(0)
+
+        frame_nb = 0
```

Comment:
This might overflow at some point. I suggest have a frame count that is reset everytime a frame is analyzed